### PR TITLE
Add sample interface and DCM

### PR DESCRIPTION
**Pull request by miagdp**

### CODE REVIEW

File: samples/EnvironmentalSensor.interface.json [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+          "schema": "long"
+        }
+      },
+      "responseSchema": "string",
+      "commandType": "synchronous"
+    },
+    {
+      "@type": "Command",
+      "name": "turnon",
+      "responseSchema": "string",
```
Comment (congkun-ck) [inline samples/EnvironmentalSensor.interface.json]: >responseSchema": "string", [](start = 7, length = 26)

This is not aligned with current DTDL schema. it should be
"response": {
    "name": "commandResponse",
    "schema": "string"
}

Similar for other requests and responses.

Comment (miagdp) [reply samples/EnvironmentalSensor.interface.json]: Pushed the updates. Please check again.

File: samples/EnvironmentalSensor.interface.json [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        "schema": "long"
+      },
+      "response": {
+        "name": "blinkResponse",
+        "schema": {
+          "@type": "Object",
+          "fields": [
+            {
+              "name": "description",
+              "schema": "string"
```
Comment (congkun-ck) [inline samples/EnvironmentalSensor.interface.json]: are you sure? it is string in your previous iteration but now it is an object.

Comment (miagdp) [reply samples/EnvironmentalSensor.interface.json]: I took from John's sample. @jspaith Can you confirm?

**Pull request merged**